src/uct/ib/base/ib_iface.c

```diff
@@ -205,6 +205,28 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Reverse Service level. 'auto' will set the same value of sl\n",
    ucs_offsetof(uct_ib_iface_config_t, reverse_sl), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"SEND_PRE_OVERHEAD", UCS_VALUE_AUTO_STR,
+   "Time spent to prepare message request and pass it to the hardware or\n"
+   "system software layers", 0,
+   UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME_UNITS,
+       {"common", "common overhead for all operation types",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.pre)},
+       {"bcopy", "additional overhead to allocate send descriptor for\n"
+                 "buffered operation",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.pre_bcopy)},
+       {NULL},)},
+
+  {"SEND_POST_OVERHEAD", UCS_VALUE_AUTO_STR,
+   "Time spent after the message request has been passed to the hardware or\n"
+   "system software layers and before operation has been finalized", 0,
+   UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME_UNITS,
+       {"common", "common overhead related to doorbell write effect on CPU\n"
+                  "operations pipeline",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.post)},
+       {"zcopy", "additional overhead to complete zero-copy operation",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.post_zcopy)},
```

Comment:
maybe have a separate config
"CQE_OVERHEAD", "Estimated overhead of processing a work request completion"
"DB_OVERHEAD", "Estimated overhead of writing a doorbell to PCI"

or like this:
SEND_OVERHEAD="wqe:40ns,bcopy:20ns,cqe:40ns,db:60ns"
wqe - posing a wqe
bcopy - allocate tx buffer
cqe - polling a cqe
db - posting a doorbell
(these components are NOT mutually exclusive)

Reply:
Replaced by one configuration option: `SEND_OVERHEAD`.